modules/rosetta/src/main/scala/org/tessellation/rosetta/server/model/RosettaSearch.scala

```diff
@@ -0,0 +1,20 @@
+package org.tessellation.rosetta.server.model
+
+import org.tessellation.schema.transaction.Transaction
+
+object RosettaSearch {
+
+  def searchBySourceAddress(transactionHash: String, snapshots: List[Transaction]) =
+    snapshots.filter(_.source == transactionHash)
```

Comment:
1. This function is pretty much an alias to `filter`
2. It compares a property of type `Address` with a `String` parameter named `transactionHash`

Reply:
This object is unneeded. Removed.